### PR TITLE
Add section for optionally bootstrapping nodes with community-provide…

### PR DESCRIPTION
**Pull request by kphed**
Similar to [this PR](https://github.com/celo-org/celo-monorepo/pull/3755), but with our Baklava node enode and the Baklava version of the proxy's container startup command.

### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Tested

Kickstarted the block-syncing process for a new proxy node using the command specified in the optional section.

### CODE REVIEW

**Comment by kphed** [comment]
@cmcewen @asaj I'm going to solicit community node contributions from other validators in the Discord server to avoid concerns over us being a central point of failure or attack vector. If it's cool with you both, I can incentivize contributions by promising a small text tag (such as their validator group name) crediting them below the section.

What do you think?

File: packages/docs/getting-started/running-a-validator-in-baklava.md [hunk 1]
```diff
@@ -216,6 +216,23 @@ export PROXY_ADDRESS=<PROXY-PUBLIC-ADDRESS>
...
 ```
 
+**OPTIONAL**
+In addition to the bootnode enode URLs provided by the cLabs team, the Celo validator community will contribute full node peers to help kickstart your nodes' peer discovery and block synchronisation processes. Here is the current list, set to `COMMUNITY_ENODES`:
+```bash
+export COMMUNITY_ENODES="enode://7c34cb0b3b9924b21e3c99134a5d46ca8242f4ecc9f722aa548ff80861be246268f1f7d8de815583a970993a7157bb37b3b5478b219c44e05888a659cb22675f@34.193.36.54:30303"
+```
+
+To use the nodes, set them to the value of the `bootnodes` or `bootnodesv4` options when running your node-starting `docker run...` commands (in this example, we will use `bootnodesv4` to keep the original command intact). To bootstrap your proxy node, you can run the command below (the same command at line 216 appended with `--bootnodesv4 $COMMUNITY_ENODES`):
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-baklava.md:225]:
```suggestion
To use the community bootnodes, set the URL above to the value of the `bootnodes` or `bootnodesv4` flags when starting your nodes using the `docker run...` command. For example, to bootstrap your proxy node, you can run the command below (the same command as earlier appended with `--bootnodesv4 $COMMUNITY_ENODES`):
```

File: packages/docs/getting-started/running-a-validator-in-baklava.md [hunk 2]
```diff
@@ -216,6 +216,23 @@ export PROXY_ADDRESS=<PROXY-PUBLIC-ADDRESS>
 docker run --name celo-proxy -it --restart unless-stopped -p 30303:30303 -p 30303:30303/udp -p 30503:30503 -p 30503:30503/udp -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --nousb --syncmode full --proxy.proxy --proxy.proxiedvalidatoraddress $CELO_VALIDATOR_SIGNER_ADDRESS --proxy.internalendpoint :30503 --etherbase $PROXY_ADDRESS --unlock $PROXY_ADDRESS --password /root/.celo/.password --allow-insecure-unlock --bootnodes $BOOTNODE_ENODES --ethstats=<YOUR-VALIDATOR-NAME>@baklava-celostats-server.celo-testnet.org
 ```
 
+**OPTIONAL**
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-baklava.md:219]:
```suggestion
**OPTIONAL**

```

File: packages/docs/getting-started/running-a-validator-in-baklava.md [hunk 3]
```diff
@@ -216,6 +216,23 @@ export PROXY_ADDRESS=<PROXY-PUBLIC-ADDRESS>
...
+```bash
+export COMMUNITY_ENODES="enode://7c34cb0b3b9924b21e3c99134a5d46ca8242f4ecc9f722aa548ff80861be246268f1f7d8de815583a970993a7157bb37b3b5478b219c44e05888a659cb22675f@34.193.36.54:30303"
+export PROXY_ADDRESS=<PROXY-PUBLIC-ADDRESS>
+docker run --name celo-proxy -it --restart unless-stopped -p 30303:30303 -p 30303:30303/udp -p 30503:30503 -p 30503:30503/udp -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --nousb --syncmode full --proxy.proxy --proxy.proxiedvalidatoraddress $CELO_VALIDATOR_SIGNER_ADDRESS --proxy.internalendpoint :30503 --etherbase $PROXY_ADDRESS --unlock $PROXY_ADDRESS --password /root/.celo/.password --allow-insecure-unlock --bootnodes $BOOTNODE_ENODES --ethstats=<YOUR-VALIDATOR-NAME>@baklava-celostats-server.celo-testnet.org --bootnodesv4 $COMMUNITY_ENODES
+```
+
+Hint: If you are running into trouble peering with the full nodes, one of the first things to check is whether your container's ports are properly configured (i.e. specifically, `-p 30303:30303 -p 30303:30303/udp` - which is set in the proxy node's command, but not the account node's command).
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-baklava.md:232]:
```suggestion
{% hint style="info" %}
If you are running into trouble peering with the full nodes, one of the first things to check is whether your container's ports are properly configured (i.e. specifically, `-p 30303:30303 -p 30303:30303/udp` - which is set in the proxy node's command, but not the account node's command).
```

File: packages/docs/getting-started/running-a-validator-in-baklava.md [hunk 4]
```diff
@@ -216,6 +216,23 @@ export PROXY_ADDRESS=<PROXY-PUBLIC-ADDRESS>
...
+
```
Comment (pranaymohan) [inline packages/docs/getting-started/running-a-validator-in-baklava.md:233]:
```suggestion
```

**Review by pranaymohan — COMMENTED** [review]
Nice add @kphed ! cc @timmoreton to take a look as well :)

**Comment by kphed** [comment]
> Nice add @kphed ! cc @timmoreton to take a look as well :)

Thanks a ton for the feedback, @pranaymohan 🙏 . I'll go over and integrate them (or ask questions, if any) this evening

Edit: Nvm, this PR will be closed.

**Comment by timmoreton** [comment]
Hey @kphed I love the idea but I think we need to figure out how this is going to work when we release 1.0.1.  We made a [change](https://github.com/celo-org/celo-blockchain/commit/98bf3225bed89b0f94b6fce2bb346650f65a328e) that means we have hardcoded defaults for bootnodes for alfajores, baklava and mainnet in the codebase (so in future you won't need to specify this).  That means that when you add `--bootnodes` to the command line, it overrides that default. Maybe we add you to that default list in `bootnodes.go`?

**Comment by kphed** [comment]
> Hey @kphed I love the idea but I think we need to figure out how this is going to work when we release 1.0.1. We made a [change](https://github.com/celo-org/celo-blockchain/commit/98bf3225bed89b0f94b6fce2bb346650f65a328e) that means we have hardcoded defaults for bootnodes for alfajores, baklava and mainnet in the codebase (so in future you won't need to specify this). That means that when you add `--bootnodes` to the command line, it overrides that default. Maybe we add you to that default list in `bootnodes.go`?

Thanks a lot for sharing those details @timmoreton. Based on that info, I will...
- Update the RC1 docs to remove the optional community nodes step that was merged last week
- Create a new PR with our RC1 and Baklava nodes added in `bootnodes.go` (changes will be tested by building + running the clients from source).
    - Would you feel comfortable with having bootnodes from other validators before 1.0.1 is released and properly tested? If so, I can post a message in the Discord server and ask for community node contributions.

If I'm missing anything, please lmk. Thanks again 🙏 .

**Comment by kphed** [comment]
> Hey @kphed I love the idea but I think we need to figure out how this is going to work when we release 1.0.1. We made a [change](https://github.com/celo-org/celo-blockchain/commit/98bf3225bed89b0f94b6fce2bb346650f65a328e) that means we have hardcoded defaults for bootnodes for alfajores, baklava and mainnet in the codebase (so in future you won't need to specify this). That means that when you add `--bootnodes` to the command line, it overrides that default. Maybe we add you to that default list in `bootnodes.go`?

Hey @timmoreton , sorry for taking a bit to get on these PRs, we've been getting hammered. The PRs below should do the trick. Thanks again for the suggestion.
- celo-blockchain PR [#1049](https://github.com/celo-org/celo-blockchain/pull/1049): Adds mainnet and Baklava nodes to the client's bootnodes lists
- celo-monorepo PR [#3924](https://github.com/celo-org/celo-monorepo/pull/3924): Removes the optional community nodes setup section

**Pull request closed without merging**